pkg/apis/vpc/v1alpha1/vpcnetworkconfiguration_types.go

```diff
@@ -20,6 +20,10 @@ type VPCNetworkConfigurationSpec struct {
 	// +optional
 	VPC string `json:"vpc,omitempty"`
 
+	// NSX path of the shared Subnets the Namespace is associated with.
+	// +optional
+	Subnets []string `json:"subnets,omitempty"`
```

Comment:
If it only refers to shared subnets, why not name it directly to SharedSubnets?

Reply:
we have shared field in Subnet CR status to show if the subnet is shared.